client/src/components/PostModal.js

```diff
@@ -0,0 +1,70 @@
+import React, { useState } from 'react';
+import { useDispatch } from 'react-redux';
+import { tweetNewMessage } from '../redux/actions/twitterActions';
+
+const PostModal = () => {
+  const [postMessage, setPostMessage] = useState('');
+  const dispatch = useDispatch();
+
+  return (
+    <div>
+      {/* the button to open tweet modal */}
+      <button
+        type="button"
+        className="btn btn-primary"
+        data-bs-toggle="modal"
+        data-bs-target="#postModal"
+        onClick={() => setPostMessage('')}
+      >
+        Create Tweet
+      </button>
```

Comment:
This could be a component on it's own and use it through your entire app and this will help you with the consistent design of your webapp.

Reply:
created ModalButton component